client/my-sites/plugins/plans/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { useTranslate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
+import MainComponent from 'calypso/components/main';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import useScrollAboveElement from 'calypso/lib/use-scroll-above-element';
+import PluginsNavigationHeader from '../plugins-navigation-header';
+
+const Plans = () => {
+	const { referenceRef: navigationHeaderRef } = useScrollAboveElement();
+
+	const translate = useTranslate();
+
+	return (
+		<MainComponent wideLayout>
+			<PageViewTracker path={ '/plugins/plans' } title={ translate( 'Plugins > Plan Upgrade' ) } />
```

Comment:
```suggestion
			<PageViewTracker path="/plugins/plans" title="Plugins > Plan Upgrade" />
```
Not sure if we should be translating these - is there prior art where this is done? (I'm assuming the title ends up in tracks, we probably want the english version for comparing results using the title - if we do that)